src/components/Comments/Comments.jsx

```diff
@@ -1,6 +1,7 @@
 import Section from '../Section/Section'
```

Comment:
Staraj się robić importy w ten sposób: 
1.najpierw importy bibliotek zewnętrznych
2.potem importy componentów
3.impoty styli
4.pozostałe importy

Każdy podpunkt oddzielasz enterem. Ogólnie jest taka zasada kolejności i przez to kod jest bardziej czytelny :D 

Podaje przykład: 

// Biblioteki zewnętrzne
import React from 'react';
import ReactDOM from 'react-dom';

// Komponenty z plików wewnętrznych
import Header from './Header';
import Sidebar from './Sidebar';

// Stylowanie
import './styles.css';

// Inne pliki i zasoby
import logo from './logo.png';

Reply:
Zazwyczaj na to zwracam uwagę ale wczoraj było już późno więc już wrzuciłem PR żeby był :D

Reply:
Spox :D